compiler/qsc_partial_eval/src/lib.rs

```diff
@@ -1181,12 +1182,16 @@ impl<'a> PartialEvaluator<'a> {
             );
             None
         };
-        let (args, ctls_arg) = self.resolve_args(
-            (store_item_id.package, callable_decl.input).into(),
-            args_value.clone(),
-            ctls,
-            fixed_args,
-        );
+        let (args, ctls_arg) = self
+            .resolve_args(
+                (store_item_id.package, callable_decl.input).into(),
+                args_value.clone(),
+                ctls,
+                fixed_args,
+            )
+            .map_err(|()| {
```

Comment:
Feels odd that we are returning an error of `()` rather than creating a real error kind, then having to map the empty error here

Reply:
The alternative is to pass `args_span` through to the underlying `resolve_args` function. I'll try that version to see if the ergonomics are better.